app/serializers/subscription_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
I don't understand what the purpose of this class is? It isn't linked into the code in any way (it only has unit tests)

Reply:
@starswan it was left over from when there was an API for administering subscriptions: but that has been descoped so this can be removed 👍